torch/distributed/nn/api/remote_module.py

```diff
@@ -30,6 +32,13 @@ def _create_module(module_cls, args, kwargs, module_interface_cls=None):
     return rpc.RRef(module, module_interface_cls)
 
 
+def _param_rrefs(module_rref, recurse):
+    def param_rref_iterator():
+        for param in module_rref.local_value().parameters(recurse):
+            yield rpc.RRef[param]
```

Comment:
What is the reason to use a generator here if we're anyways returning a list? We can just create and append to the list directly?

Reply:
Avoided the iterator here.